src/components/trendingBooks/BookCard.tsx

```diff
@@ -0,0 +1,14 @@
+import Link from '../../../node_modules/next/link';
```

Comment:
The below suggestion should work? Were there issues?
```suggestion
import Link from "next/link";
```

Reply:
No issues, I think my auto import is just doing this sometimes and I don't always notice. Fixed